phpstan.neon.dist

```diff
@@ -5,6 +5,7 @@ includes:
 parameters:
     level: max
     checkMissingIterableValueType: false
+    treatPhpDocTypesAsCertain: false
```

Comment:
Since some of the code is a bit pedantic despite the docblock values, this means we already don't trust the user input

Reply:
isn't it more because they can't assert that people using guzzle also have phpstan or related, so they can't rely on phpdoc as a safeguard ?